src/util/nodemap/build_nodemap.h

```diff
@@ -497,9 +510,15 @@ static inline int MPIR_NODEMAP_build_nodemap(int sz,
         out_nodemap[i] = j;
     }
 
-odd_even_cliques:
-    if (odd_even_cliques)
-    {
+cliques:
+    if (MPIR_CVAR_NUM_CLIQUES > 1) {
+        for (i = 0; i < sz; ++i) {
+            if (i % MPIR_CVAR_NUM_CLIQUES) {
+                out_nodemap[i] += g_max_node_id + (i % MPIR_CVAR_NUM_CLIQUES );
```

Comment:
I see what you're saying. That wasn't my intention to have those overlapping. I'll have to fix that.